index.html

```diff
@@ -236,7 +236,20 @@ <h2>skills</h2>
       <hr>
       <br>
     </div>   
-  
+  </section>
+   
+  <section class="contact" id="contact">
+    <div class="contact_wrapper">
+      <h2 class="section-header">Contact me</h2>
+      <p class="description">If you have an application you are interested in developing, 
+        a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- Hey, great adding the [formspree](https://formspree.io/), however, you need to get a unique id and add it to the URL in order to have your form to be submitted correctly. please, kindly fix this issue to improve the user experience of your website.